package.json

```diff
@@ -33,6 +33,7 @@
     "LICENSE"
   ],
   "dependencies": {
+    "chai": "^4.2.0",
```

Comment:
@medtoure18 any reason why should we add this lib? 🤔

Reply:
@PeterNgTr because tests wasn't running until i added that dependency

Reply:
have you tried to put it under package.json of your test repo?

Reply:
No because i didn't need it.
I can remove it if you prefer, i was just thinking about future contributors.

Reply:
just asking. But nothing impacts. So it's fine for me.

Reply:
thanks